rpcclient/errors.go

```diff
@@ -411,7 +411,10 @@ var BtcdErrMap = map[string]error{
 	"transaction already exists in blockchain":    ErrTxAlreadyConfirmed,
 
 	// A transaction in the mempool.
-	"already have transaction in mempool": ErrTxAlreadyInMempool,
+	//
+	// NOTE: For btcd v0.24.2 and beyond, the error message is "already
```

Comment:
shouldn't this be `For btcd versions prior to v0.24.2, the error message is "already have transaction in mempool"`?

Reply:
In `v0.24` the error msg is `already have transaction`, in `0.24.2` the error msg is changed to `already have transaction in mempool`, otherwise the match won't fail since `already have transaction` is a subset of `already have transaction in mempool`.

Reply:
Thanks now it makes sense.